playbook.yml

```diff
@@ -22,8 +22,9 @@
     when: sonar == "db"
 
```

Comment:
Please explain this part with a comment. And if its a bug, then please report to Ansible and add it here.

Reply:
waiting for permissions to open the bug on Bugzilla